app/javascript/dashboard/routes/dashboard/settings/automation/constants.js

```diff
@@ -132,6 +132,13 @@ export const AUTOMATIONS = {
         inputType: 'multi_select',
         filterOperators: OPERATOR_TYPES_1,
       },
+      {
```

Comment:
let's call the attribute `conversation_language` itself for consistency. `current_language` might be confusing to the end users